src/pydap/tests/test_responses_ascii.py

```diff
@@ -53,24 +51,25 @@ def test_headers(self):
 
     def test_body(self):
         """Test the generated ASCII response."""
-        self.assertEqual(self.res.text, """Dataset {
-    Sequence {
-        String id;
-        Int32 lon;
-        Int32 lat;
-        Int32 depth;
-        Int32 time;
-        Int32 temperature;
-        Int32 salinity;
-        Int32 pressure;
-    } cast;
-} SimpleSequence;
----------------------------------------------
-cast.id, cast.lon, cast.lat, cast.depth, cast.time, cast.temperature, cast.salinity, cast.pressure
-"1", 100, -10, 0, -1, 21, 35, 0
-"2", 200, 10, 500, 1, 15, 35, 100
-
-""")
+        self.assertEqual(self.res.text,
```

Comment:
You are right that it is much less readable. I guess my idea here was that since it is a unittest, readability is important but what is more important is exactitude. This way, one sees precisely what characters are expected and extraneous spaces at EOLs are not hidden. We can easily turn off flake8 to that whole file, if you prefer.